fm/templates/fm/modal.html

```diff
@@ -20,11 +20,10 @@
                         <div class="modal-body"></div>
                         <div class="modal-buttons">
                             {% block modal_buttons %}
-                            <button class="btn btn-primary" type="submit">
-                                {% trans 'OK' %}
+                            <button class="btn btn-primary modal-ok" type="submit" data-fm-default-ok="{% trans 'OK' %}">
                             </button>
                             <button data-dismiss="modal" class="btn btn-default" type="button"><i class="fa fa-times"></i>
-                                {% trans 'Cancel' %}
+                                <span class="modal-cancel" data-fm-default-cancel="{% trans 'Cancel' %}"></span>
```

Comment:
Why we need span element here?

Reply:
To allow changing the value of the label without changing the icon (storing the icon html and forcing users to set it in data attribute wouldn't be optimal). And also I needed to get the translated default from somewhere, and the data attribute was the best solution that I was able to find.

Reply:
Then I think we have to wrap OK label in similar span and use it for selectors. Just for symmetry. Also I suggest to rename `modal-cancel` to `modal-cancel-label`. And the same for `modal-ok` if we add extra span for it. Does this make sense?